### PR TITLE
Hide --sdk-version and several workload-list options

### LINKED ISSUE #25871: Hide `--sdk-version` and other parameters to workload commands
From https://github.com/dotnet/sdk/issues/20357:

> The `--sdk-version` parameter to workload commands is intended for VS for Mac to use in order to download workloads before a new version of the .NET SDK is installed.

> It isn't intended to be used from the command line, and it doesn't work for MSI-based installs.  So we should hide it from the list of options displayed in help.

> We should also remove this from the documentation

From https://github.com/dotnet/sdk/pull/20387#issuecomment-909405390

> this PR hides the `--sdk-version` option (which is present on most of the dotnet workload commands) from the help, so it should be removed from the docs as well. 
> I also hid all of the options on workload list besides verbosity and help in [this PR](https://github.com/dotnet/sdk/pull/20271), so if you could make those updates as well that would be great. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2bbfaa61-ebac-d118-6dd1-152a53f37a91
* Version Independent ID: 55bbba16-ae98-5abc-cd42-1b677105fd29
* Content: [dotnet workload install command - .NET CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-workload-install)
* Content Source: [docs/core/tools/dotnet-workload-install.md](https://github.com/dotnet/docs/blob/main/docs/core/tools/dotnet-workload-install.md)
* Product: **dotnet-fundamentals**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #25871

### CODE REVIEW

File: docs/core/tools/dotnet-workload-list.md [hunk 1]
```diff
@@ -44,8 +41,6 @@ For more information about the `dotnet workload` commands, see the [dotnet workl
 
 [!INCLUDE [no-cache](../../../includes/cli-no-cache.md)]
```
Comment (gewarren) [inline docs/core/tools/dotnet-workload-list.md:42]: Should this and some of the others be removed?

File: docs/core/tools/dotnet-workload-update.md [hunk 2]
```diff
@@ -62,8 +62,6 @@ For more information about the `dotnet workload` commands, see the [dotnet workl
 
 [!INCLUDE [no-cache](../../../includes/cli-no-cache.md)]
 
-[!INCLUDE [sdk-version](../../../includes/cli-sdk-version.md)]
-
 [!INCLUDE [source](../../../includes/cli-source.md)]
```
Comment (gewarren) [inline docs/core/tools/dotnet-workload-update.md:65]: Remove source and temp-dir includes?

**Review by gewarren — APPROVED** [review]

File: docs/core/tools/dotnet-workload-update.md (hunk 2 above)
Comment (tdykstra) [reply docs/core/tools/dotnet-workload-update.md:65]: Those were just moved in the synopsis, they're still valid options.

File: docs/core/tools/dotnet-workload-list.md (hunk 1 above)
Comment (tdykstra) [reply docs/core/tools/dotnet-workload-list.md]: Yes, good catch, thanks!

**Review by dsplaisted — APPROVED** [review]

**Comment by sfoslund** [comment]
@tdykstra sorry, I missed this in my original review of the PR, but we also hid the  [--download-to-cache <CACHE>] [--from-cache <CACHE>] options, can we remove those from the docs as well?

**Comment by tdykstra** [comment]
And --no-cache ?

**Comment by sfoslund** [comment]
No, no cache is a nuget option so that's still there. Only those two workload caching options.

**Pull request merged**